src/index.rs

```diff
@@ -20,6 +20,7 @@ use {
 
 mod entry;
 mod rtx;
+mod tx_fetcher;
```

Comment:
```suggestion
mod fetcher;
```

Reply:
I was actually looking at extending this struct to fetch blocks, hashes, headers, and block count. Maybe `HyperJsonRpc` is a better name?